src/pages/ProjectStart/Components/CardItem.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+function CardItem(props) {
+  const { title, redTitle, buttonText } = props;
+  return (
+    <CardItemWrapper>
+      <header>
+        <ItemTitle>{title}</ItemTitle>
+      </header>
+      <ButtonDiv>
```

Comment:
```suggestion
      <ButtonWrapper>
```
컴포넌트 이름에 태그 명이 들어가는게 좀 어색해보여서 다른 이름 제안 드립니다

Reply:
수정했습니다!